src/App/Link.cpp

```diff
@@ -179,7 +179,7 @@ void LinkBaseExtension::setProperty(int idx, Property *prop) {
 }
 
 App::DocumentObjectExecReturn *LinkBaseExtension::extensionExecute(void) {
-    // The actual value of LinkRecompouted is not important, just to notify view
+    // The actual value of LinkRecomputed (obsolete) is not important, just to notify view
```

Comment:
This typo originally corresponded to a token by this corrected name, but that has since been removed.